runestone/chapterdb/dbchapterinfo.py

```diff
@@ -13,13 +13,15 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
+# **************************************************************
+# |docname| - Record chapter info in the database during a build
+# **************************************************************
 
 __author__ = "bmiller"
 
-import os.path
+from typing import Sequence
```

Comment:
Sequence is deprecated in Python 3.9

Reply:
Thanks. From what I can tell, the new replacement of `from collections.abc import Sequence` [requires Python 3.9](https://www.python.org/dev/peps/pep-0585/). Do we want to support Python 3.8 or earlier versions? We might want to think about some sort of policy. Per https://endoflife.date/python, Python 3.6 is (essentiall) at its end of life; Python 3.7 is EOL in 1 1/2 years; Python 3.8 is EOL in ~3 years.